src/pyrva_talk/twitter/api.py

```diff
@@ -7,19 +7,19 @@
 
 
 auth = get_auth()
-api = tweepy.API(auth)
-py_rva = api.get_user("@PyRVA")
+twitter_api = tweepy.API(auth)
+py_rva = "@PyRVA"
```

Comment:
This went from whatever api.get_user() returns to just a string, and I don't see any changes where this gets used later. Was that an existing bug that was fixed?

Reply:
@stephen1000 and I discovered that the `api.get_user()` function was not appropriate to pass as the default value for the functions defined in the api.